modules/ROOT/pages/installation/osx.adoc

```diff
@@ -66,3 +66,13 @@ server.jvm.additional=-XX:-MaxFDLimit
 Without this setting, the file descriptor limit for the JVM will not be increased beyond 10240.
 Note, however, that this only applies to macOS.
 On all other operating systems, you should always leave the `MaxFDLimit` JVM setting enabled.
+
+== Uninstall Neo4j
+
+In case you want to specifically remove Neo4j Desktop, refer to https://neo4j.com/docs/desktop-manual/current/troubleshooting/re-install/[Neo4j Desktop -> Re-installation of Desktop].
+
+Here are the steps to uninstall Neo4j on macOS:
+
+. (Optional) Create a xref:/backup-restore/index.adoc[backup] to avoid losing your data.
+. Delete the _Neo4j Desktop.app_ from the Applications folder.
+. Delete the _~/Library/Application Support/Neo4j Desktop_ and _~/.neo4j_ folders.
```

Comment:
earlier in the page we tell mac users to install with tar, so this will need the same instructions as that. I don't know if these extra steps about deleting something from the Applications folder are necessary, maybe check with someone who has a mac. @davidoliverSP2 perhaps?